.github/workflows/linux.yml

```diff
@@ -0,0 +1,85 @@
+name: Linux with OpenJDK 1.8 and OpenJDK 11 GitHub CI
```

Comment:
Everything looks good, just have a hesitation about this title... it's long and makes it harder to find at a glance a particular build in the GitHub PR checks:

![image](https://user-images.githubusercontent.com/325433/100430877-a61c4e80-3097-11eb-8b11-db1c2e31a31e.png)

Just calling it "Linux" or "Linux GitHub CI" would probably be enough, the name of the job is added at the end and clarifies which bit of the build is executed. E.g. may look like:

``Linux GitHub CI / openjdk11 (pull request)"